core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -278,11 +279,9 @@ object ConfigCommand extends Config {
     val nl = System.getProperty("line.separator")
     val addConfig = parser.accepts("add-config", "Key Value pairs of configs to add. Square brackets can be used to group values which contain commas: 'k1=v1,k2=[v1,v2,v2],k3=v3'. The following is a list of valid configurations: " +
             "For entity_type '" + ConfigType.Topic + "': " + nl + LogConfig.configNames.map("\t" + _).mkString(nl) + nl +
-            "For entity_type '" + ConfigType.Broker + "': " + nl + KafkaConfig.dynamicBrokerConfigs.map("\t" + _).mkString(nl) + nl +
-            "For entity_type '" + ConfigType.Client + "': " + nl + "\t" + QuotaConfigOverride.ProducerOverride
-                                                            + nl + "\t" + QuotaConfigOverride.ConsumerOverride + nl +
-            "For entity_type '" + ConfigType.User + "': " + nl + "\t" + QuotaConfigOverride.ProducerOverride
-                                                          + nl + "\t" + QuotaConfigOverride.ConsumerOverride + nl +
+            "For entity_type '" + ConfigType.Broker + "': " + nl + DynamicConfigs.Broker.names().map("\t" + _).mkString(nl) + nl +
+            "For entity_type '" + ConfigType.User + "': " + nl + DynamicConfigs.Client.names().map("\t" + _).mkString(nl) + nl +
+            "For entity_type '" + ConfigType.Client + "': " + nl + DynamicConfigs.Client.names().map("\t" + _).mkString(nl) +
```

Comment:
`mkString` allows you to specify `start`, `sep`, `end`. I think that can simplify this code a little. Also, you may consider using string interpolation.